analyses/cell-types-annotation/README.md

```diff
@@ -0,0 +1,65 @@
+# Pipeline for annotating cell types for sc-/sn-RNA-Seq Analysis in 10X Genomics data
+
+## Usage
+
+To run the script in this module from the command line sequentially, use:
+
+```
+bash run-cell-types-annotation.sh
+```
+
+`run-cell-types-annotation.sh` is designed to be run as if it was called from this module directory even when called from outside of this directory.
+
+Parameters according to the project and analysis strategy will need to be specified in the following scripts:
+- `project_parameters.Config.yaml` located at the `root_dir`
+- `run-cell-types-annotation.R`: define `root_dir`, `data_dir`, and define params for: `01-cell-types-annotation-SingleR-broad.Rmd` and `02-cell-types-annotation-SingleR-fine.Rmd` in the `rmarkdown::render code chunk`
+
+
+## Folder content
+This folder contains a script tasked to identify cell types annotation with SingleR by using a broad and fine resolution labeling across the project.
+
+########----------------------------------------------------------------------------------------------------------------
+## TODO based on discussion on July 1st, 2024
```

Comment:
Just double checking, were these notes meant to be left in the public version of the README?

Reply:
Nope, there shouldn't be there 😅 Fixed here a77e41362c6f9a39f3cd25be045271052fdb72d2.